cloud/services/compute/loadbalancers.go

```diff
@@ -211,10 +215,12 @@ func (s *Service) DeleteLoadbalancers() error {
 		name := path.Base(*s.scope.Network().APIServerTargetProxy)
 		op, err := s.targetproxies.Delete(s.scope.Project(), name).Do()
 		if err != nil {
-			return errors.Wrapf(err, "failed to delete forwarding rules")
+			if !gcperrors.IsNotFound(err) {
```

Comment:
Similar to above, this conditional can be combined with the `err != nil` check above